src/actions/breakpoints/addBreakpoint.js

```diff
@@ -61,7 +61,8 @@ export default async function addBreakpoint(
     location: newLocation,
     astLocation,
     hitCount,
-    generatedLocation: newGeneratedLocation
+    generatedLocation: newGeneratedLocation,
+    text: breakpoint.text
```

Comment:
This is temporary; should be removed once https://github.com/devtools-html/debugger.html/pull/5823 is merged.